### ISSUE TITLE
Actual onboarding

### ISSUE THREAD

**Issue by thatoddmailbox** [MyHomeworkSpace/client#119]
We probably want to have an actual onboarding thing of some sort. (hopefully not just a tour with a lot of reading and clicking) Even just a simple thing explaining how to add a class would be good, as right now we just dump a new user onto the Homework page, and they have to somehow figure out where the Classes page is hiding before they can do anything.

**Comment by amazansky** [comment]
Bumping this because it feels important to me—also this would be a good place to let the user know about the many handy keyboard shortcuts!

**Comment by willbarkoff** [comment]
I'm not sure what the best way to do this would be. @thatoddmailbox makes a good point about a lot of reading and clicking, but I don't think a modal with a wall of text would necessarily be the way to go either. @amazansky do you have any suggestions? In terms of keyboard shortcuts, #136 could also be relevant.

**Comment by thatoddmailbox** [comment]
I think probably we want to at least have something for adding/changing the default classes, since right now that's very hidden and you really have to do that before anything else.

Apart from that, maybe just something to point them at Quick Add? I guess it really depends on how detailed we want to go.

**Comment by amazansky** [comment]
Yeah, something pointing them to adjust the default classes, showing them how the tag system works, showing them keyboard shortcuts (and the **?** shortcut for them to be able to see all the shortcuts at any time), and introducing quick add seems like a good place to start. Maybe we could take it from there.

In terms of presentation, I agree with @willbarkoff that a modal is not the way to go. Maybe we could try something a bit more interactive.
